app/code/Magento/Checkout/view/frontend/web/js/view/shipping-information.js

```diff
@@ -8,7 +8,8 @@ define([
     'uiComponent',
     'Magento_Checkout/js/model/quote',
     'Magento_Checkout/js/model/step-navigator',
-    'Magento_Checkout/js/model/sidebar'
+    'Magento_Checkout/js/model/sidebar',
+    'Magento_Checkout/js/view/shipping'
```

Comment:
> Some components listed below uses calls like stepNavigator.isProcessed('shipping').
This call means that this components have shipping component as dependence and expected that it already exists and initialized, but Magento_Checkout/js/view/shipping.js not listed first argument array for define()-function. Seems that with shipping-component in requirejs dependencies this components would be rendered when shipping-step was already registered in step-navigator.

Do we really need to inject this dependency?
I guess we should be able to use the corresponding RequireJS properties for this purpose.

https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/javascript/requirejs_concept.html#requirejs-configuration-file

Reply:
Are you mean `deps` RequireJS property? If yes, as I understand from requirejs docs, this property should be used for js components that not supports AMD loader, and where we can't add manualy dependency to their source code. Therefore I think in this case it would be better to keep dependencies in define() first argument.

Reply:
Hi @likemusic, sorry for late reply and thanks for your patience!
I meant the `shim` configuration node, but you are right, it's also related to non-AMD scripts.

Please note, there are still a couple of automated test failures with the current solution:
https://travis-ci.org/magento/magento2/jobs/510436396
https://travis-ci.org/magento/magento2/jobs/510436404